### PR TITLE
feat: scrollbar-language-dialog

### LINKED ISSUE #191: Show indicator (scrollbar or arrow) in languages
Currently if you open settings and either languages or currency there is no indicator there are actually more options available and is not intuitive so either show scrollbar or some arrow at the end to indicate there are mor3 options.

### PR DESCRIPTION
**Pull request by alexbarnsley**
Scrollbar on select options in settings screen

resolves #191

### CODE REVIEW

File: src/assets/styles/layout.scss [hunk 1]
```diff
@@ -37,3 +37,7 @@ form {
   color: color($colors, primary, base);
   -webkit-text-stroke: .4px color($colors, primary, base);
 }
+
+.alert-md.select-alert div[ng-reflect-ng-switch] {
```
Comment (luciorubeens) [inline src/assets/styles/layout.scss]: `.alert-md`? this will only change on Android.

Comment (alexbarnsley) [reply src/assets/styles/layout.scss]: It's not specific to Android? alert-md is an Angular class I believe. You can see it in chrome:

image

Comment (alexbarnsley) [reply src/assets/styles/layout.scss]: I'll change it to `ion-alert` - i tried that first but it didn't work (which is why i used `alert-md`). Just tried again and it is so bare with

Comment (alexbarnsley) [reply src/assets/styles/layout.scss]: Updated 👌

**Pull request merged**